testing/src/test/java/io/stargate/it/backends/AllBackendSuite.java

```diff
@@ -36,4 +43,7 @@
 
   @Parameterized.Parameter(2)
   public String version;
+
+  @Parameterized.Parameter(3)
+  public Integer numberOfStargateNodes;
```

Comment:
I think that this parameter would be useful for other persistence implementations.